src/jetstream/plugins/kubernetes/auth/basic_auth.go

```diff
@@ -75,5 +76,13 @@ func (c *KubeBasicAuth) GetUserFromToken(cnsiGUID string, cfTokenRecord *interfa
 }
 
 func (c *KubeBasicAuth) RegisterJetstreamAuthType(portal interfaces.PortalProxy) {
-	// Not needed - 'creds' is built-in
+	// Register auth type with Jetstream - use the same as the HttpBasic auth
+
+	auth := c.portalProxy.GetAuthProvider(authTypeHttpBasic)
+	if auth.Handler != nil {
+		c.portalProxy.AddAuthProvider(c.GetName(), interfaces.AuthProvider{
```

Comment:
I still get the same error. Had a quick look and got stuck when I realised I think it should be using `(c *KubernetesSpecification) AddAuthProvider` instead of `(p *portalProxy) AddAuthProvider` (it's failing in `(c *KubernetesSpecification) GetAuthProvider`.